elements/lisk-db/src/kv_store.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import * as path from 'path';
+import * as fs from 'fs';
+import { debug } from 'debug';
+import encodingDown from 'encoding-down';
```

Comment:
Do we need this if lisk-codec is going to be implemented?

Reply:
when we integrate codec, this needs to be removed along with changing the interface to Buffer

Reply:
OK maybe leave a comment?